test/ng/directive/ngOptionsSpec.js

```diff
@@ -2143,6 +2143,25 @@ describe('ngOptions', function() {
       expect(scope.selected).toEqual([]);
     });
 
+    it('should be possible to specify a value for empty options', function() {
+      var html = '<select ng-model="someModel" ng-options="c for c in choices" ng-empty-value="2+2">' +
```

Comment:
I think the attribute should reference a scope property. This is more in line with what people will do.